opg-php-fpm-71-ppa-1604/docker/confd/templates/app.conf.tmpl

```diff
@@ -0,0 +1,102 @@
+{{   if exists "/opg/nginx/ssl/force/redirect" }}
+server {
+    listen      80;
+    server_name {{ getv "/opg/nginx/server/names" }};
+    return      307 https://$host$request_uri;
+}
+{{   end }}
+
+
+server {
+    #Using global cert let's also listen on 443
+    listen     443 ssl;
+    {{ if exists "/opg/nginx/server/names" }}
+    server_name {{ getv "/opg/nginx/server/names" }};
+    {{ end }}
+    {{ if exists "/opg/nginx/server/ssl/protocols" }}
+    ssl_protocols {{ getv "/opg/nginx/server/ssl/protocols" }};
+    {{ else }}
+    ssl_protocols TLSv1.2 TLSv1.1 TLSv1;
+    {{ end }}
+    {{ if exists "/opg/nginx/server/ssl/ciphers" }}
+    ssl_ciphers {{ getv "/opg/nginx/server/ssl/ciphers" }};
+    {{ else }}
+    ssl_ciphers HIGH:!aNULL:!MD5;
+    {{ end }}
+    rewrite_log  on;
+    access_log  /var/log/app/nginx.access.json logstash_json;
+    error_log   /var/log/app/nginx.error.log error;
+
+
+# Defaults to /app/public if OPG_NGINX_ROOT isn't set
+    {{ if exists "/opg/nginx/root" }}
+    set $root "{{ getv "/opg/nginx/root" }}";
+    {{ else }}
+    set $root "/app/public";
+    {{ end }}
+
+    root $root;
+
+    location =/meta.json {
+        alias /app/META;
+    }
+
+    # i.e. deny access to .htaccess in case someone will create it
+    location ~ /\. {
+        deny  all;
+    }
+
+    # Add HTTP Strict Transport Security header - required for .service.gov.uk domains
+    add_header Strict-Transport-Security "max-age=31536000; includeSubDomains; preload";
+    # Prevent clickjacking
+    add_header X-Frame-Options "SAMEORIGIN";
+    add_header X-XSS-Protection "1; mode=block";
+    add_header X-Content-Type-Options nosniff;
+
+    # serve static files directly
+
+# Defaults to index.html if OPG_NGINX_INDEX isn't set
+    {{ if exists "/opg/nginx/index" }}
+    set $index "{{ getv "/opg/nginx/index" }}";
+    {{ else }}
+    set $index "index.php";
+    {{ end }}
+
+    index $index;
+
+    #We've found some php client libraries not understanding chunked response
+    chunked_transfer_encoding off;
+
+ #    fastcgi_buffers 256 4k;
```

Comment:
If we are commenting this, Why not remove it?

Reply:
this one also is coming from the php-fpm image, just checked on vitor's (original container) and it seems to be commented out on our current setups as well. will remove them shortly

Reply:
Done!